### PR TITLE
Reorganize HPA page based on latest releases

### PR DESCRIPTION
**Pull request by superseb**
https://github.com/rancher/docs/issues/829

@rawmind0 can you double-check the changes here?

### CODE REVIEW

File: content/rancher/v2.x/en/k8s-in-rancher/horitzontal-pod-autoscaler/_index.md [hunk 1]
```diff
@@ -295,7 +222,7 @@ For HPA to use custom metrics from Prometheus, package [k8s-prometheus-adapter](
 
 #### Assigning Additional Required Roles to Your HPA
```
Comment (rawmind0) [inline content/rancher/v2.x/en/k8s-in-rancher/horitzontal-pod-autoscaler/_index.md]: Additional roles are not needed from rancher v2.0.7, due to not using anymore kubelet `read-only-port` and using k8s certificates instead`system:anonymous` user.

Section `Assigning Additional Required Roles to Your HPA` should be moved to `Manual Installation`

File: content/rancher/v2.x/en/k8s-in-rancher/horitzontal-pod-autoscaler/_index.md [hunk 2]
```diff
@@ -295,7 +222,7 @@ For HPA to use custom metrics from Prometheus, package [k8s-prometheus-adapter](
...
 
-By default, HPA reads resource and custom metrics with the user `system:anonymous`. Assign `system:anonymous` the the `view-resource-metrics` and `view-custom-metrics` in the ClusterRole and ClusterRoleBindings manifests. These roles are used to access metrics. 
+By default, HPA reads resource and custom metrics with the user `system:anonymous`. Assign `system:anonymous` the the `view-resource-metrics` and `view-custom-metrics` in the ClusterRole and ClusterRoleBindings manifests. These roles are used to access metrics.
```
Comment (rawmind0) [inline content/rancher/v2.x/en/k8s-in-rancher/horitzontal-pod-autoscaler/_index.md]: Fix typo

```
- By default, HPA reads resource and custom metrics with the user `system:anonymous`. Assign `system:anonymous` the the `view-resource-metrics` and `view-custom-metrics` in the ClusterRole and ClusterRoleBindings manifests. These roles are used to access metrics.
+ By default, HPA reads resource and custom metrics with the user `system:anonymous`. Assign `system:anonymous` to `view-resource-metrics` and `view-custom-metrics` in the ClusterRole and ClusterRoleBindings manifests. These roles are used to access metrics.
```

**Comment by superseb** [comment]
@rawmind0 Moved the roles section to manual install and corrected the typo

**Comment by rawmind0** [comment]
Thanks @superseb ...

LGTM

**Pull request merged**